src/SDKs/Subscription/Management.Subscription/Generated/Models/Page1.cs

```diff
@@ -0,0 +1,53 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.Subscription.Models
+{
+    using Microsoft.Rest;
+    using Microsoft.Rest.Azure;
+    using Newtonsoft.Json;
+    using System.Collections;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Defines a page in Azure responses.
+    /// </summary>
+    /// <typeparam name="T">Type of the page content items</typeparam>
+    [JsonObject]
+    public class Page1<T> : IPage<T>
```

Comment:
anything we can do about this?

Reply:
If not, no worries. Looks like several other SDKs have the same thing going on.

Reply:
this is expected